Cargo.toml

```diff
@@ -19,6 +19,15 @@ members = [
     "crates/codegen/testing",
     "crates/infra/cli",
     "crates/infra/utils",
+
+    "crates/testlang/inputs/language",
```

Comment:
nit: I suggest keeping the `members` entries sorted alphabetically.

Reply:
This was because testlang is more 'core' than solidity, but done.